### PR TITLE
Create ddns-ipv64.xml

### PR DESCRIPTION
**Pull request by TerrierX**
This Docker container is a DDNS updater for Free DynDNS - ipv64.net

### CODE REVIEW

File: ddns-ipv64.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Project>https://github.com/alcapone1933/docker-ddns-ipv64</Project>
+  <Overview>This Docker container is a DDNS updater for Free DynDNS - ipv64.net.&#xD;
+&#xD;
+If the ipv4 address at the site is changed, the new ipv4 address is sent to ipv64.net as an A-Record.</Overview>
+  <Category>Tools: Network:DNS</Category>
+  <Icon>https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQasFPd0QKbbYpPvuUG0EXhoVFF-Kw2XcdqDg&amp;amp;s</Icon>
+  <Requires>List of all Parameters that can be configured https://github.com/alcapone1933/docker-ddns-ipv64?tab=readme-ov-file#env-parameter</Requires>
+  <Config Name="Timezone" Target="TZ" Default="Europe/Berlin" Mode="" Description="Timezone" Type="Variable" Display="always" Required="true" Mask="false"/>
```
Comment (nwithan8) [inline ddns-ipv64.xml:15]: A `TZ` timezone variable automatically gets passed into every Docker container on Unraid, so no need to include it in the configuration.
```suggestion
```

File: ddns-ipv64.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="Polling frequenz Domain" Target="CRON_TIME_DIG" Default="*/30 * * * *" Mode="" Description="Polling frequenz for the domain (dig DOMAIN_IPV64 A)" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Domain(s) ipv64" Target="DOMAIN_IPV64" Default="" Mode="" Description="YOUR DOMAINS: e.g. your-domain.ipv64.net, your-domain.ipv64.de" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Domain Key" Target="DOMAIN_KEY" Default="" Mode="" Description="DOMAIN KEY: YOUR DOMAIN KEY or DynDNS update hash can be found at https://ipv64.net/dyndns.php" Type="Variable" Display="always" Required="true" Mask="true"/>
+  <Config Name="Domain Präfix yes|no" Target="DOMAIN_PRAEFIX_YES" Default="yes|no" Mode="" Description="DOMAIN PRAEFIX YES: This means that the domain PRAEFIX is actively used" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Domain Präfix" Target="DOMAIN_PRAEFIX" Default="" Mode="" Description="YOUR DOMAIN PRAEFIX (subdomain) : &#x26A0;&#xFE0F; Use only one prefix &#x26A0;&#xFE0F; e.g. ddns" Type="Variable" Display="always" Required="false" Mask="false"/>
```
Comment (nwithan8) [inline ddns-ipv64.xml:21]: Not sure if emojis will render properly in the Unraid GUI.

```suggestion
  <Config Name="Domain Präfix" Target="DOMAIN_PRAEFIX" Default="" Mode="" Description="YOUR DOMAIN PRAEFIX (subdomain). Use only one prefix (e.g. ddns)" Type="Variable" Display="always" Required="false" Mask="false"/>
```

File: ddns-ipv64.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="IP check" Target="IP_CHECK" Default="yes|no" Mode="" Description="IP CHECK: The IP address of the domain is checked" Type="Variable" Display="always" Required="false" Mask="false"/>
```
Comment (nwithan8) [inline ddns-ipv64.xml:22]: Fields with a dropdown picker should be marked Required, with the default action first in the pipe list.

```suggestion
  <Config Name="IP check" Target="IP_CHECK" Default="yes|no" Mode="" Description="IP CHECK: The IP address of the domain is checked" Type="Variable" Display="always" Required="true" Mask="false"/>
```

File: ddns-ipv64.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="Shoutrrr URL" Target="SHOUTRRR_URL" Default="" Mode="" Description="SHOUTRRR URL: Your Shoutrrr URL as notification service e.g. ( gotify,discord,telegram,email)" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Shoutrrr URL Skip Test" Target="SHOUTRRR_SKIP_TEST" Default="" Mode="" Description="SHOUTRRR_SKIP_TEST: No test message is sent when the container is started" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Name Server" Target="NAME_SERVER" Default="ns1.ipv64.net" Mode="" Description="NAME_SERVER: : The name server to check the IP address of your domain" Type="Variable" Display="always" Required="false" Mask="false"/>
```
Comment (nwithan8) [inline ddns-ipv64.xml:25]:
```suggestion
  <Config Name="Name Server" Target="NAME_SERVER" Default="ns1.ipv64.net" Mode="" Description="NAME_SERVER: : The name server to check the IP address of your domain" Type="Variable" Display="always" Required="false" Mask="false">ns1.ipv64.net</Config>
```

File: ddns-ipv64.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="Location logs and script" Target="/data" Default="/mnt/user/appdata/ddns-ipv64" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false"/>
```
Comment (nwithan8) [inline ddns-ipv64.xml:26]:
```suggestion
  <Config Name="Location logs and script" Target="/data" Default="/mnt/user/appdata/ddns-ipv64" Mode="rw" Description="" Type="Path" Display="advanced-hide" Required="false" Mask="false">/mnt/user/appdata/ddns-ipv64</Config>
```

Config paths for containers stored in the "appdata" folder are typically put in the "advanced" section, as they don't require immediate set-up by the user (given they have a valid default). Marking is as "advanced-hide" will prevent users from being able to delete this config value from the GUI (force them to set it for their own sake).

**Review by nwithan8 — CHANGES_REQUESTED** [review]

**Comment by TerrierX** [comment]
@nwithan8 Thank you for the useful hints that I weren't aware of

File: ddns-ipv64.xml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Project>https://github.com/alcapone1933/docker-ddns-ipv64</Project>
+  <Overview>This Docker container is a DDNS updater for Free DynDNS - ipv64.net.&#xD;
+&#xD;
+If the ipv4 address at the site is changed, the new ipv4 address is sent to ipv64.net as an A-Record.</Overview>
+  <Category>Tools: Network:DNS</Category>
+  <Icon>https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQasFPd0QKbbYpPvuUG0EXhoVFF-Kw2XcdqDg&amp;amp;s</Icon>
+  <Requires>List of all Parameters that can be configured https://github.com/alcapone1933/docker-ddns-ipv64?tab=readme-ov-file#env-parameter</Requires>
+  <Config Name="Polling frequency IP address" Target="CRON_TIME" Default="*/15 * * * *" Mode="" Description="Polling frequency of the current IP address" Type="Variable" Display="always" Required="true" Mask="false"/>
```
Comment (nwithan8) [inline ddns-ipv64.xml:15]: Sorry, forgot about these defaults in my last review.
```suggestion
  <Config Name="Polling frequency IP address" Target="CRON_TIME" Default="*/15 * * * *" Mode="" Description="Polling frequency of the current IP address" Type="Variable" Display="always" Required="true" Mask="false">*/15 * * * *</Config>
```

File: ddns-ipv64.xml [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  <Config Name="Polling frequenz Domain" Target="CRON_TIME_DIG" Default="*/30 * * * *" Mode="" Description="Polling frequenz for the domain (dig DOMAIN_IPV64 A)" Type="Variable" Display="always" Required="true" Mask="false"/>
```
Comment (nwithan8) [inline ddns-ipv64.xml:16]:
```suggestion
  <Config Name="Polling frequenz Domain" Target="CRON_TIME_DIG" Default="*/30 * * * *" Mode="" Description="Polling frequenz for the domain (dig DOMAIN_IPV64 A)" Type="Variable" Display="always" Required="true" Mask="false">*/30 * * * *</Config>
```

File: ddns-ipv64.xml [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+  <Config Name="Domain(s) ipv64" Target="DOMAIN_IPV64" Default="" Mode="" Description="YOUR DOMAINS: e.g. your-domain.ipv64.net, your-domain.ipv64.de" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Domain Key" Target="DOMAIN_KEY" Default="" Mode="" Description="DOMAIN KEY: YOUR DOMAIN KEY or DynDNS update hash can be found at https://ipv64.net/dyndns.php" Type="Variable" Display="always" Required="true" Mask="true"/>
+  <Config Name="Domain Präfix yes|no" Target="DOMAIN_PRAEFIX_YES" Default="yes|no" Mode="" Description="DOMAIN PRAEFIX YES: This means that the domain PRAEFIX is actively used" Type="Variable" Display="always" Required="false" Mask="false"/>
```
Comment (nwithan8) [inline ddns-ipv64.xml:19]:
```suggestion
  <Config Name="Domain Präfix yes|no" Target="DOMAIN_PRAEFIX_YES" Default="yes|no" Mode="" Description="DOMAIN PRAEFIX YES: This means that the domain PRAEFIX is actively used" Type="Variable" Display="always" Required="true" Mask="false"/>
```

File: ddns-ipv64.xml [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+  <Config Name="IP check" Target="IP_CHECK" Default="yes|no" Mode="" Description="IP CHECK: The IP address of the domain is checked" Type="Variable" Display="always" Required="true" Mask="false"/>
+  <Config Name="Shoutrrr URL" Target="SHOUTRRR_URL" Default="" Mode="" Description="SHOUTRRR URL: Your Shoutrrr URL as notification service e.g. ( gotify,discord,telegram,email)" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Shoutrrr URL Skip Test" Target="SHOUTRRR_SKIP_TEST" Default="" Mode="" Description="SHOUTRRR_SKIP_TEST: No test message is sent when the container is started" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="Name Server" Target="NAME_SERVER" Default="ns1.ipv64.net" Mode="" Description="NAME_SERVER: : The name server to check the IP address of your domain" Type="Variable" Display="always" Required="false" Mask="false">ns1.ipv64.net</Config>
+  <Config Name="Location logs and script" Target="/data" Default="/mnt/user/appdata/ddns-ipv64" Mode="rw" Description="" Type="Path" Display="advanced-hide" Required="false" Mask="false">/mnt/user/appdata/ddns-ipv64</Config>
```
Comment (nwithan8) [inline ddns-ipv64.xml:25]:
```suggestion
  <Config Name="Location logs and script" Target="/data" Default="/mnt/user/appdata/ddns-ipv64" Mode="rw" Description="" Type="Path" Display="advanced-hide" Required="true" Mask="false">/mnt/user/appdata/ddns-ipv64</Config>
```

**Review by nwithan8 — CHANGES_REQUESTED** [review]
Sorry, forgot a few things in my last review. After this, it looks good to go from me.

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**